templates/projects/_helpers.html

```diff
@@ -369,6 +376,67 @@ <h2><strong> {{_('Overall progress')}}</strong>: {{ overall_progress | round | i
         {% endif %}
     </div>
     {% endif %}
+    <script>
+      const toggleButton = document.getElementById("toggle-button");
+
+      const new_task_a_tag_admin = document.getElementById("new_task_admin");
+      const new_task_a_tag_user = document.getElementById("new_task_user");
+      const new_task_a_tag_overall = document.getElementById("new_task_overall");
```

Comment:
minor. as this is anchor tags, would it make code more readable if we name them as `a_tag_new_task_admin` ...

Reply:
Makes sense. Will update.